blueprint/src/loops.tex

```diff
@@ -114,37 +117,45 @@ \section{Preliminaries}
   where we use $k_{i_0} = 0$ in the first equality.
 \end{proof}
 
-\begin{lemma}
-\label{lem:refined_caratheodory}
-  If a point $x$ of $E$ lies in the convex hull of a set $P$, then $x$
-  can be written as the convex combination of at most $d + 1$ affinely
-  independent points in $P$ with positive coefficients.
-\end{lemma}
-
-\begin{proof}
-  \uses{lem:caratheodory}
-  \cref{lem:caratheodory} gives points $p_i$ in $P$ and non-negative
-  weights $w_i$ such that $x = \sum w_i p_i$ for $0 ≤ i ≤ k$ where
-  $k ≤ d$. We first discard every point with vanishing weights.
-  Now suppose the $p_i$'s are not affinely independent. Then there is
-  some relation $p_{i_0} = \sum λ_j p_j$ with positive weights
-  $λ_j$. We can then discard $p_{i_0}$ and replace each remaining
-  $w_j$ by $w_j + w_{i_0}λ _j$.
-\end{proof}
-
 
 \begin{definition}
   \label{def:surrounds_points}
   \lean{surrounding_pts}
   \leanok
-  Let $F$ be a real vector space with dimension $d$. A point $x$ in $E$
+  A point $x$ in $E$
   is surrounded by points $p_0$, \dots, $p_d$ if those points are
   affinely independent and there exist weights $w_i ∈ (0, 1)$ such
   that $x = \sum_i w_i p_i$.
 \end{definition}
 
 Note that, in the above definition, the number of points $p_i$ is fixed
-by the dimension $d$ of $F$.
+by the dimension $d$ of $E$, and that the weights $w_i$ are the barycentric
+coordinates of $x$ with respect to the affine basis $p_0, \ldots, p_d$.
+The following lemma provides a convenient alternate characterisation of
+being surrounded.
```

Comment:
I don't see how you can call that lemma a "characterisation of being surrounded". The characterisation you have in mind is rather a corollary of the lemma: "x is surrounded by $p_0$, \dots, $p_d$ if those points are affinely independent and x is the interior of their convex hull".